src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -127,14 +129,18 @@ export const ShortTerm = ({
                                 ))}
                             </div>
                         </div>
-                        <Instructions instructions={instructions} useV4Design={useV4Design} />
                         {isPreapproved && (
                             <PreapprovalDisclaimer
                                 preapprovalDisclaimerBody={preapprovalDisclaimerBody}
                                 preapprovalDisclaimerHeadline={preapprovalDisclaimerHeadline}
                                 country={country}
```

Comment:
```suggestion
                                country={country}
                                useNewCheckoutDesign={useNewCheckoutDesign}
```

If we could pass in `useNewCheckoutDesign` to the `PreapprovalDisclaimer` we could then tweak some of the styles based on the fact that the new checkout modal is going to be narrower than how it is today. If we could then add a `checkout` class [here](https://github.com/surekhaw/paypal-messaging-components/blob/f4c6ed370edd9b498aa32c5ae662e9e13c9a5ec3/src/components/modal/v2/parts/PreapprovalDisclaimer.jsx#L7):
```
<div className={`content__row preapproval-disclaimer ${useNewCheckoutDesign === 'true' ? 'checkout' : ''}`}>
```

Then we can update the style in `_preapproval-disclaimer` under [this line](https://github.com/surekhaw/paypal-messaging-components/blob/f4c6ed370edd9b498aa32c5ae662e9e13c9a5ec3/src/components/modal/v2/styles/components/_preapproval-disclaimer.scss#L127):
```
    &.checkout {
        h2 {
            margin-left: 0px;
        }

        ul {
            padding-left: 5px;
        }
    }
```